package.json

```diff
@@ -130,9 +130,8 @@
     "express": "^4.17.1",
     "express-pino-logger": "^6.0.0",
     "express-winston": "^4.2.0",
-    "fastify": "1.14.6",
-    "fastify2": "npm:fastify@2.15.3",
-    "fastify3": "npm:fastify@3.21.5",
+    "fastify": "4.24.3",
```

Comment:
Can we add the caret please? I don't remember why the 1.14 version wasn't making use of the caret 🤔 Maybe it was a mistake